main.js

```diff
@@ -0,0 +1,220 @@
+    conditions: data.weather[0].main,
+    icon: `https://openweathermap.org/img/wn/${data.weather[0].icon}@4x.png`
+  })
+
+  renderWeather();
+};
+
+var renderWeather = function() {
+  document.querySelector('.current-weather').replaceChildren();
+
+  for (let i = 0; i < currentWeather.length; i++) {
+    const weather = currentWeather[i];
+
+    var template = `
+    <div class='col-md-4'>
+      <div class='weather'>
+        <h1>${weather.temp}º</h1>
+        <h2>${weather.name}</h2>
+        <h3>${weather.conditions}</h3>
+      </div>
+    </div>
+    <div class='col-md-4'>
+      <div class='icon'>
+        <img src=${weather.icon}>
+      </div>
+    </div>`;
+
+    document.querySelector('.current-weather').insertAdjacentHTML('beforeend', template);
+  }
+};
+
+// Get 5 day forecast
+
+var fetchPlace = function(city) {
+  const url = 'http://api.openweathermap.org/geo/1.0/direct?q=';
+  const apiKey = '&appid=0e79198cac257d0139d13e84e8617759'
+  const fullUrl = url + city + apiKey;
+  fetch(fullUrl, {
+    method: 'GET',
+    dataType: 'json'
+  })
+  .then(data => data.json())
+  .then(data => fetchForecast(data));
+};
+
+var fetchForecast = function(data) {
+  const apiKey = '0e79198cac257d0139d13e84e8617759'
+  const url = `http://api.openweathermap.org/data/2.5/forecast?lat=${data[0].lat}&lon=${data[0].lon}&appid=${apiKey}&units=imperial`;
+
+  fetch(url, {
+    method: 'GET',
+    dataType: 'json'
+  })
+  .then(data => data.json())
+  .then(data => addForecast(data));
+};
+
+var addForecast = function (data) {
+  const dayOne = [];
+  const dayTwo = [];
+  const dayThree = [];
+  const dayFour = [];
+  const dayFive = [];
+
+  dayOne.push({
+    conditions: data.list[0].weather[0].main,
+    temp: Math.round(data.list[0].main.temp),
+    icon: `https://openweathermap.org/img/wn/${data.list[0].weather[0].icon}@2x.png`,
+    day: formattedDates[0]
+  })
+
+  dayTwo.push({
+    conditions: data.list[7].weather[0].main,
+    temp: Math.round(data.list[7].main.temp),
+    icon: `https://openweathermap.org/img/wn/${data.list[7].weather[0].icon}@2x.png`,
+    day: formattedDates[1]
+  })
+
+  dayThree.push({
+    conditions: data.list[15].weather[0].main,
+    temp: Math.round(data.list[15].main.temp),
+    icon: `https://openweathermap.org/img/wn/${data.list[15].weather[0].icon}@2x.png`,
+    day: formattedDates[2]
+  })
+
+  dayFour.push({
+    conditions: data.list[23].weather[0].main,
+    temp: Math.round(data.list[23].main.temp),
+    icon: `https://openweathermap.org/img/wn/${data.list[23].weather[0].icon}@2x.png`,
+    day: formattedDates[3]
+  })
+
+  dayFive.push({
+    conditions: data.list[31].weather[0].main,
+    temp: Math.round(data.list[31].main.temp),
+    icon: `https://openweathermap.org/img/wn/${data.list[31].weather[0].icon}@2x.png`,
+    day: formattedDates[4]
+  })
+
+
+  weatherForecast.push({
+    dayOne, dayTwo, dayThree, dayFour, dayFive
+  })
+
+  renderForecast();
+};
+
+var renderForecast = function() {
```

Comment:
also this type of function could have been reused. You can create a more 'dumb' implementation that takes querySelector, template as params and makes the action